notes/Chapter-Decision/chap1.tex

```diff
@@ -350,17 +350,18 @@ \subsection{The utility maximization theorem}
 
 \subsection{Bayesian probability systems}
 
-We now focus on the conditional probability functions involved in Theorem \ref{ch1:utility-maximization}.
-Indeed, recall that these functions are not exactly probability functions as we know them, because we only require that they satisfy Bayes formula.
-Since, by the fundamental theorem above, these are the right tools to encapsulate the behaviour of a rational agent,
-it is important for us to understand precisely their algebraic structure.
-Let us formalize this concept by a definition.
+We now focus on the conditional probability functions involved in Theorem \ref{ch1:utility-maximization} (respecting the second condition of the Theorem). We define the notion of Bayesian probability system to name them.
```

Comment:
Theorem \ref{ch1:utility-maximization} (respecting the second condition of the Theorem) introduces a particular set of functions similar to conditional probability functions. It is through those that the agents assess the probability of occurrence of different events. Our focus in this section is on these functions, which we call Bayesian Probability Systems.